src/browser.ts

```diff
@@ -3,9 +3,17 @@ import { getApiKey } from './api-key'
 import { port } from './port'
 
 export function openInstallerInBrowser() {
+  if (process.env.NODE_ENV === 'development') {
+    return
+  }
+
   shell.openExternal(`http://localhost:${port.value}/installer/?v=${getApiKey()}`)
 }
 
 export function openDashboardInBrowser() {
+  if (process.env.NODE_ENV === 'development') {
+    return
+  }
+
```

Comment:
not saying this is wrong, but why?

Reply:
Hmmm the idea was to prevent the "auto opening" of the UI, but now thinking about it, this might not be the right place do it...